crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -67,6 +67,8 @@ module sui::sui_system {
         validator_report_records: VecMap<address, VecSet<address>>,
         /// Schedule of stake subsidies given out each epoch.
         stake_subsidy: StakeSubsidy,
+        /// Unix timestamp of the current epoch start
+        epoch_start_timestamp_ms: u64,
```

Comment:
This is probably worth exposing via an accessor (see L531 of this file for an example):

```rust
public fun epoch_start_timestamp_ms(self: &SuiSystemState): u64 {
    self.epoch_start_timestamp_ms
}
```